### ISSUE TITLE
build 10.0.17763.292 / Printers

### ISSUE THREAD

**Issue by Julien-nl** [stascorp/rdpwrap#652]
Dear all
after the update to 10.0.17763.292 [](https://github.com/stascorp/rdpwrap/issues/645) we lost the printers on the remote desktop.
before whenever we log in to the terminal we can print using the redirect printers but they are gone
can someone please advies how to get those back ?

**Comment by Julien-nl** [comment]
issue is fixed, appear to be rdpwrap does not works on no english windows,
i tested it on both english and no english windows.
no english windows does not works ( only patched .dll files works )
english windows works

**Comment by arni7** [comment]
Welcome
strange thing after adding the entry from the first post
![image](https://user-images.githubusercontent.com/47143787/51906245-0242b880-23c4-11e9-8825-7c6e378d450f.png)

but printing works, after adding the @ditchmagnet file, everything works fine except printing
Please, do you have any sugestion?

**Comment by binarymaster** [comment]
Duplicate of #645.

**Comment by Julien-nl** [comment]
i have visited the link [645](https://github.com/stascorp/rdpwrap/issues/645) however it does not talks about the printers but only about getting the RDP working with 292.
i Do have this already working no issue with the Remote desktop but printing from the Remote desktop to the Local Printer is impossible.
Can someone please advies how to get this fixed ? my .ini file as attached [https://github.com/infowayrs/INI-RDPWRAP/blob/master/10.0.17763.292.zip](url)

**Comment by bulgaru** [comment]
@ditchmagnet, @binarymaster 
Currently on 1809 (build 17763.379).
The output of the Conf is:
<img width="403" alt="Screenshot 2019-03-13 at 11 29 50 AM" src="https://user-images.githubusercontent.com/9161540/54268211-5e802700-4583-11e9-9b93-dadead3f0a3a.png">

Confirm the bug described by @arni7. Indeed, the Printer redirection does not work. Steps to confirm that it's RDP Wrapper related:

- installed latest RDP Wrapper
- updated the ini with the latest version for 1809 (17763.292 build). Everything working except printer redirection
- uninstalled RDP completely, but enabled Remote Desktop Access
- when accessing without the RDP Wrapper, the redirection works perfectly.

The conclusion is that termserv.dll is ok and should work fine. 
I will try debugging the issue further and will keep you updated on the results.

**Comment by bulgaru** [comment]
Hello, progress.

I took the patched termserv.dll from [here](https://www.mysysadmintips.com/windows/clients/545-multiple-rdp-remote-desktop-sessions-in-windows-10).

I removed the RDP Wrapper entirely after substituting the dll. The Remote Desktop connections should be re-allowed from the settings.

Printer redirection works perfectly. I assume something is off with the settings and needs further investigation. Either way - it's a quick solution for those blocked by the printers issue.

**Comment by Mindaugas85** [comment]
> Hello, progress.
> 
> I took the patched termserv.dll from [here](https://www.mysysadmintips.com/windows/clients/545-multiple-rdp-remote-desktop-sessions-in-windows-10).
> 
> I removed the RDP Wrapper entirely after substituting the dll. The Remote Desktop connections should be re-allowed from the settings.
> 
> Printer redirection works perfectly. I assume something is off with the settings and needs further investigation. Either way - it's a quick solution for those blocked by the printers issue.

IT doesn't work.
Change key values with Tiny Hexar on termsrv.dll
Add values to rdpwrap.ini to support 10.0.19041.746 and after a restart - printers are disappearing, but multisession is working.
